tests/integration/project/project_test_scenario.go

```diff
@@ -0,0 +1,20 @@
+package project
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+func ProjectTestScenario() {
+	When("Executing Machine readable output tests", func() {
+		It("Help for odo project list should contain machine output", HelpCommand)
+		It("should be able to get project", GetProject)
+		It("should display the help", HelpCommandProject)
+		It("should display only the project name", CommandWithQFlag)
+		It("--wait should work with deleting a project", CommandWaitFlag)
```

Comment:
This one has nothing to do with machine-readable output either. It should be in a different spec, IMO.